DataFormats/Detectors/TRD/src/Tracklet64.cxx

```diff
@@ -43,6 +43,35 @@ GPUd() int Tracklet64::getPadCol() const
   return CAMath::Nint(6.f + mcmCol * ((float)NCOLMCM) + padLocal);
 }
 
+GPUd() float Tracklet64::PadPositionMCM() const
+{
+  return 12.0 - (getPositionBinSigned() * GRANULARITYTRKLPOS);
```

Comment:
`getPositionBinSigned() * GRANULARITYTRKLPOS` should give the local pad number relative to the MCM center. Why are you subtracting 12 here? Shouldn't it be 10.5? But in any case I am not sure if these functions are needed

Reply:
From my memory, the reason for 12 is that it matches the digits in the raw data display. At the time, I did not try to figure out why this works.

Reply:
in `UncalibratedPad()` there is also a +2 which I don't understand. This effectively turns the 12 into a 10, so closer to the expected 10.5